### PR TITLE
Draft a potential project to replace fedc with renovate

### PR DESCRIPTION
**Pull request by wjt**
As discussed at FOSDEM

### CODE REVIEW

**Comment by wjt** [comment]
@razzeee Does this roughly match what you expect this project would involve?

File: README.md [hunk 1]
```diff
@@ -99,3 +99,17 @@ If we could keep the database (but move it to postgres) and also keep the api (p
 Possible mentor: @razzeee
 
 Difficulty: Hard
+
+## Replace Flatpak External Data Checker with Renovate
+
+[Flatpak External Data Checker](https://github.com/flathub-infra/flatpak-external-data-checker) is a tool maintained and operated by the Flathub team to help keep apps on Flathub up-to-date. Using annotations in the build manifest for each app, it monitors for new releases of each app and its bundled dependencies, and opens pull requests as needed to update them.
+
+While this tool has been useful in helping Flathub scale to its current size, it has limitations. It tends to open many overlapping pull requests when several modules of an application are updated at once; the build manifest annotations it consumes are somewhat ad-hoc and idiosyncratic; there are limits to what it can check and update; and and so on.
```
Comment (razzeee) [inline README.md:143]: Plus the checker can't be run on gitlab. (talked with a KDE person on the way back home and it seemed like this would help them)

**Comment by razzeee** [comment]
I think we will need those data sources and a manager. I've glanced over https://github.com/renovatebot/renovate/blob/main/docs/development/adding-a-package-manager.md and it does seem to be rather simple, but seems to assume, that it can call out to a binary that does the actual "lock file upgrade".

So we would need to write some kind of comandline tool to  do that too, is my guess.

**Comment by razzeee** [comment]
Opened https://github.com/renovatebot/renovate/discussions/27087 on renovates side of things

**Pull request merged**